src/com/aca/Nare_Silanyan/homework1/Client.java

```diff
@@ -0,0 +1,53 @@
+package com.aca.Nare_Silanyan.homework1;
+
+import com.aca.Nare_Silanyan.homework1.models.Manager;
+import com.aca.Nare_Silanyan.homework1.models.Parameters;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class Client {
+    public static void main(String[] args) {
+
+        Parameters pen_Video = new Parameters("Pen",4,new String[]{"Video"}, new String[]{"MP4", "VGA"},null, 2,6500 );
+        Parameters pen_Record = new Parameters("Pen",5, new String[]{"Record"}, new String[]{"MP3"},"8GB", 9,14000 );
```

Comment:
avoid of constructors with many parameters, use the Builder pattern to create an object instead of this way

Reply:
Should be discussed.